Smith_Ben/Assignments/Ruby/OOP/Game/samurai.rb

```diff
@@ -0,0 +1,41 @@
+
+require_relative 'human'
+
+class Samurai < Human
+	@@num_of_samurais = 0
+	def initialize
+		super
+		@health = 200
+		@@num_of_samurais += 1
+		puts
+		self
+	end
+
+	def death_blow(obj)
+		while obj.health > 0
+			obj.health -= 1
```

Comment:
Thats a pretty weak death blow if it only takes away 1 health.  That's more of a slap_blow.